config/vite.config.ts

```diff
@@ -47,7 +47,16 @@ export default defineConfig({
     }),
   ],
   server: {
-    allowedHosts: true,
+    // Vite 7 requires explicit allowedHosts for security
+    // Allow localhost, local network access, and test domains
+    allowedHosts: [
+      'localhost',
+      '.localhost',
```

Comment:
1. This cannot work. 
2. It will not be able to be run by any domain. It's too limiting. 
3. There must allow all hosts. We run talawa on multiple domains and end users will too